pkgs/servers/pleroma/mix.nix

```diff
@@ -281,6 +281,19 @@ let
       beamDeps = [ ecto ];
     };
 
+    crypt = buildRebar3 rec {
+      name = "crypt";
```

Comment:
```suggestion
      pname = "crypt";
```

Reply:
Is it automatically generated? If it is, then, I think something else should change.